scripts/start.js

```diff
@@ -74,11 +77,12 @@ function setupCompiler (port) {
     clearConsole();
     const hasErrors = stats.hasErrors();
     const hasWarnings = stats.hasWarnings();
+
     if (!hasErrors && !hasWarnings) {
       console.log(chalk.green('Compiled successfully!'));
       console.log();
       // eslint-disable-next-line
-      console.log('The app is running at http://localhost:' + port + '/');
+      console.log('The app is running at http://localhost:' + port + basePath);
```

Comment:
yeah thats my bad, i didnt really get time to set up the dev experience when i made this (was a very recent change :))